### PR TITLE
feat(normalizeItemType): solution type normalization, normalizedType …

### PR DESCRIPTION
**Pull request by drewctate**
…in content

AFFECTS PACKAGES:
@esri/hub-common
@esri/hub-content

ISSUES CLOSED: #367, #366

### CODE REVIEW

**Review by benstoltz — APPROVED** [review]

File: packages/content/src/content.ts [hunk 1]
```diff
@@ -29,6 +30,12 @@ export function getContent(
       return Promise.reject(e);
     });
   }
+  return ret.then(content => applyNormalizedType(content));
```
Comment (tomwayson) [inline packages/content/src/content.ts:33]: this should be done in `itemToContent()`, not here

**Review by tomwayson — COMMENTED** [review]
please make the change I've requested before releasing

File: packages/content/src/content.ts (hunk 1 above)
Comment (drewctate) [reply packages/content/src/content.ts:33]: Won't that make it so that it's only attached when we're using the portal API?

Comment (tomwayson) [reply packages/content/src/content.ts:33]: nope, `itemToContent()` is called by `datasetToContent()` as well

Comment (drewctate) [reply packages/content/src/content.ts:33]: Nevermind. I see

**Pull request merged**